packages/cadl-ranch-specs/http/enums/fixed/main.cadl

```diff
@@ -0,0 +1,40 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+
+@scenarioService("/fixed-enums")
+namespace FixedEnums;
+
+@fixed
+enum DaysOfWeekEnum {
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+  "Sunday",
+}
+
+@route("/string")
+interface String {
+  @scenario
+  @scenarioDoc("Expect to handle a known value. Mock api will return 'Monday'")
+  @get
+  @route("/known-value")
+  getKnownValue(): DaysOfWeekEnum;
+
+  @scenario
+  @scenarioDoc("Expect to send a known value. Mock api expect to receive 'Monday'")
+  @put
+  @route("/known-value")
+  putKnownValue(@body body: DaysOfWeekEnum): void;
+
+  @scenario
+  @scenarioDoc("Expect to handle an unknown value. Mock api expect to receive 'Weekend'")
+  @put
+  @route("/unknown-value")
+  putUnknownValue(@body body: DaysOfWeekEnum): void;
```

Comment:
Is that a scenario that is possible with fixed enums? I coudl understad the reverse, a service actually returning a new value but sending an unknown value should that work?

Reply:
Well, in that case I think it would be a client failure... it there a way to model that in cadl-ranch?

Reply:
I mean you could just model it as the service returning this unknown value and then the generator test needs to do whatever it wants to handle that(try catch the exception for example)

Maybe we  could have another decorator that says that this scenario is really to handle a case that is bad and fail nicely

Reply:
@iscai-msft is there something you recommend here?